config.spec.d.ts

```diff
@@ -354,6 +354,11 @@ export interface Schema {
    */
   "filepath_template_raster"?: string;
 
+  /**
+   * Disable visualization of geo or aggregate data for this schema?
+   */
+  "disabled_mode"?: "geo" | "aggregate";
```

Comment:
Cool; I like the simplicity. I think I'd prefer "disable_mode" versus "disabled_mode." To me that more clearly conveys the intent.